### PR TITLE
work on print()-methods and parameter-filtering

### PR DESCRIPTION
**Pull request by strengejacke**

### CODE REVIEW

File: R/utils_clean_stan_parameters.R [hunk 1]
```diff
@@ -25,7 +25,7 @@
 
 #' @keywords internal
 .clean_parameters <- function(x) {
-  removers <- grep("^(prior_|sd_|cor_|lp__)", x$Parameter)
+  removers <- grep("^(prior_|sd_|cor_|lp__|smooth_sd)", x$Parameter)
```
Comment (DominiqueMakowski) [inline R/utils_clean_stan_parameters.R]: Shouldn't we deal such name cleaning in insight?

File: R/rope.R [hunk 2]
```diff
@@ -62,45 +62,35 @@ as.double.rope <- function(x, ...) {
...
-  }
+  cat(sprintf(
+    "# Proportion%s of samples inside the ROPE [%.2f, %.2f]:\n\n",
+    ifelse(all(x$CI[1] == x$CI), "", "s"),
+    x$ROPE_low[1],
+    x$ROPE_high[1]
+  ))
+
+  # I think this is something nobody will understand and we'll probably forget
+  # why we did this, so I'll comment a bit...
```
Comment (DominiqueMakowski) [inline R/rope.R]: BTW, thank you for having this commenting habit, it is really helpful!
I definitely have to force myself to document my code more  :)

File: R/utils_clean_stan_parameters.R (hunk 1 above)
Comment (strengejacke) [reply R/utils_clean_stan_parameters.R]: Yes, true. I'll open an issue there, and once we've implemented it in insight, we can remove the code from here...

File: R/equivalence_test.R [hunk 3]
```diff
@@ -4,9 +4,9 @@
...
+#' @details Using the \link[=rope]{ROPE} and the \link[=hdi]{HDI}, Kruschke (2018) suggest using the percentage of the 95\% (or 90\%, considered more stable) \link[=hdi]{HDI} that falls within the ROPE as a decision rule. If the HDI is completely outside the ROPE, the "null hypothesis" for this parameter is "rejected". If the ROPE completely covers the HDI, i.e. all most credible values of a parameter are inside the region of practical equivalence, the null hypothesis is accepted. Else, it’s undecided whether to accept or reject the null hypothesis. If the full ROPE is used (i.e., 100\% of the HDI), then the null hypothesis is rejected or accepted if the percentage of the posterior within the ROPE is smaller than to 1\% or greater than 99\%.
 #'
-#' @references Kruschke JK. Rejecting or Accepting Parameter Values in Bayesian Estimation. Advances in Methods and Practices in Psychological Science. 2018; doi: \doi{10.1177/2515245918771304}
+#' @references Kruschke JK. Rejecting or Accepting Parameter Values in Bayesian Estimation. Advances in Methods and Practices in Psychological Science. 2018; \doi{10.1177/2515245918771304}
```
Comment (DominiqueMakowski) [inline R/equivalence_test.R]: Should we standardize citation style  (APA norms, ...)?

Comment (strengejacke) [reply R/equivalence_test.R]: Yes, we probably could use a reference manager (I personally use zotero) to create the references or so.

Comment (DominiqueMakowski) [reply R/equivalence_test.R]: Even faster could be to use the **cite** button below the paper on [google scholar](https://scholar.google.fr/scholar?hl=fr&as_sdt=0%2C5&q=Kruschke+JK.+Rejecting+or+Accepting+Parameter+Values+in+Bayesian+Estimation.+Advances+in+Methods+and+Practices+in+Psychological+Science.+2018&btnG=) and then copy paste the APA reference

Comment (strengejacke) [reply R/equivalence_test.R]: I have those citations in zotero anyway, so it's also just a click away

Comment (DominiqueMakowski) [reply R/equivalence_test.R]: APA is ok for my perspective (guess I am used to it), but as long as it's consistent...

**Comment by strengejacke** [comment]
Yeah, you might want to try this. It's rather an extreme case, but it works!

```r
library(bayestestR)
library(brms)

zinb <- read.csv("http://stats.idre.ucla.edu/stat/data/fish.csv")

b12 <- brm(
  bf(count ~ child + camper + (1 | persons), 
     zi ~ child + camper + (1 | persons)),
  data = zinb,
  family = zero_inflated_poisson()
)
 
hdi(b12, ci = c(.5, .8, .9), effects = "all", component = "all")
```

File: R/rope.R [hunk 4]
```diff
@@ -13,17 +13,19 @@
...
-#'   standardized parameter (negligible effect size according to Cohen, 1988).
-#'   This could be generalized: For instance, for linear models, the ROPE
-#'   could be set as \code{0 +/- .1 * sd(y)}. Kruschke (2010, 2011, 2014) suggests
-#'   using the proportion of the 95\% (or 90\%, considered more stable)
-#'   \link[=hdi]{HDI} that falls within the ROPE as an index for
-#'   "null-hypothesis" testing (as understood under the Bayesian framework,
-#'   see \link[=equivalence_test]{equivalence_test}). Besides the ROPE-based
-#'   decision criteria, the proportion of the 95\% CI that falls in the ROPE
-#'   can be used as a continuous index.
+#'   purposes (\cite{Kruschke 2010, 2011, 2014}).
```
Comment (strengejacke) [inline R/rope.R]: @DominiqueMakowski I'm not sure where you are referring to, could you please add the references in APA-style here?

**Pull request merged**